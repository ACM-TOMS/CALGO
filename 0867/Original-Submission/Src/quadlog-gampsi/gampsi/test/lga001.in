lnGamma
0
0
0 0
0 0
-1.7976e+308 +1.7976e+308
0 0
0 1 0
1 1 0
2 1 0
3 1 0
4 1 0
1 0 0
0 0 0
