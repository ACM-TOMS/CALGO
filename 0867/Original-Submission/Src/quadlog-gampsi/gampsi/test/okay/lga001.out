### Numerical evaluation of ln(Gamma(x))
###
### x = f * 2^p
###
### Line format: 2 f p neval result relerr abserr
###
2                    0.     0 0                                                  Infinity 0 0
2                    1.     0 0   0.0000000000000000000000000000000000000000000000000e-01 0 0
2                    2.     0 0   0.0000000000000000000000000000000000000000000000000e-01 0 0
2                    3.     0 0   6.9314718055994530941723212145817656807550013436026e-01 0 0
2                    4.     0 0   1.7917594692280550008124773583807022727229906921830e+00 0 0
2                  Inf     0 0                                                  Infinity 0 0
2                  NaN     0 0                                                       NaN 0 0
